bugbug/model.py

```diff
@@ -402,12 +409,15 @@ def train(self, importance_cutoff=0.15):
         # Evaluate results on the test set for some confidence thresholds.
         for confidence_threshold in [0.6, 0.7, 0.8, 0.9]:
             y_pred_probas = self.clf.predict_proba(X_test)
+            confidence_class_names = self.class_names + ["Not classified"]
 
             y_test_filter = []
             y_pred_filter = []
+            not_classified_indices = []
             for i in range(0, len(y_test)):
                 argmax = np.argmax(y_pred_probas[i])
                 if y_pred_probas[i][argmax] < confidence_threshold:
+                    not_classified_indices.append(i)
                     continue
 
                 y_test_filter.append(y_test[i])
```

Comment:
We are no longer using a subset of y_test, so you can just remove y_test_filter altogether and use y_test (converting it to a string array when needed with `y_test.astype(str)`).

Reply:
There are two problems 
1. The subset of y_test is still being used for multi-label model.
2. I've appended the not-classified y_test labels to the last of y_test_filter because we're concatening in case of y_pred_filter too. To do what you're proposing, we'll have to find a way to insert "NOT_CLASSIFIED" labels at the corresponding positions and not concatenate them at the last.

Reply:
For 1, can't we support the multi-label case too?

For 2, it's pretty easy, in the loop instead of `not_classified_indices.append(i)` you append \_\_\_NOT_CLASSIFIED\_\_\_